support-frontend/assets/pages/[countryGroupId]/oneTimeCheckout.tsx

```diff
@@ -342,6 +343,10 @@ function OneTimeCheckoutComponent({
 		if (finalAmount) {
 			setIsProcessingPayment(true);
 
+			if (paymentMethod === 'None') {
```

Comment:
Out of interest is there a reason why the equivalent check on the generic checkout checking `paymentMethod === undefined` whilst this is `'None'` for `paymentMethod`? 🤔

Reply:
There's a slight inconsistency, the generic makes the default `undefined` in the setState whereas the one-time sets it as `None`

Reply:
Makes sense, possibly another point to add to the improving maintainability of the checkouts list, consistency of common types.